server.js

```diff
@@ -149,8 +157,96 @@ api.all(/(.*)/, bodyParser.json(), async (req, res) => {
   });
 });
 
+version.use("/:versionNumber", (req, res, next) => {
+  const versionNumber = req.params.versionNumber;
+
+  // Construct the directory path based on the versionNumber path segment
+  const staticDir = path.join(__dirname, "versions", versionNumber);
+
+  // Verify that the file path is under the static directory for security reasons
+  filePath = fs.realpathSync(path.resolve(staticDir));
```

Comment:
You need to declare this using either `const` or `let`, the latter being for if you intend to mutate it. Same for other similarily introduced code blocks in this file.